packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -15,30 +37,132 @@ export default util.createRule({
     messages: {
       missingAccessibility:
         'Missing accessibility modifier on {{type}} {{name}}.',
+      unwantedPublicAccessibility:
+        'Public accessibility modifier on {{type}} {{name}}.',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          accessibility: accessibilityLevel,
+          overrides: {
+            type: 'object',
+            properties: {
+              accessors: accessibilityLevel,
+              constructors: accessibilityLevel,
+              methods: accessibilityLevel,
+              properties: accessibilityLevel,
+              parameterProperties: accessibilityLevel,
+            },
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
   },
-  defaultOptions: [],
-  create(context) {
+  defaultOptions: [{ accessibility: 'explicit' }],
+  create(context, [option]) {
+    /**
+     * Reads the value set on the Override and returns a Check value
+     * Check value is used to control what, if any accessibility modifiers are required or banned
+     */
+    function parseOverride(
+      defaultCheck: AccessibilityLevel,
+      overrideToCheck?: AccessibilityLevel,
+    ): AccessibilityLevel {
+      return typeof overrideToCheck === 'undefined'
+        ? defaultCheck
+        : overrideToCheck;
+    }
+
+    let baseCheck: AccessibilityLevel = 'explicit';
```

Comment:
this code could now become the following (allows you to switch `let` for `const`)

```TS
create(context, [{ accessibility: baseCheck, overrides }]) {
  // ...

  overrides = overrides || {};
  const ctorCheck = overrides.constructors || baseCheck;
  const accessorCheck = overrides.accessors || baseCheck;
  const methodCheck = overrides.methods || baseCheck;
  const propCheck = overrides.properties || baseCheck;
  const paramPropCheck = overrides.parameterProperties || baseCheck;
}
```

Reply:
that's so much cleaner.